src/client/app/effects/github.ts

```diff
@@ -9,9 +9,10 @@ import { Effect, Actions } from '@ngrx/effects';
 import { Http, ResponseContentType } from '@angular/http';
 import * as clipboard from 'clipboard';
 import { UIEffects } from './ui';
-import { find } from 'lodash';
+import { find, isNil } from 'lodash';
```

Comment:
Do you need `isNil`? you can just do `(exception == null)` that checks for both null and undefined [literally the implementation of isNil].